### ISSUE TITLE
Could we get a way to tell the Remnant more about the Deep right after they ask for keystones?

### ISSUE THREAD

**Issue by dseomn** [endless-sky/endless-sky#9160]
### Problem Description

Each time I've gotten the mission where Ivan asks for quantum keystones, I've agreed to get them because I figure that means Ivan won't look for somebody else to do it. Which means I have time to talk to the Remnant before giving Ivan any keystones, to let the Remnant know what he's up to, and get their advice on what I should do. E.g., maybe they know a way to make fake keystones that I could give to Ivan so he gives up on that idea. But when I visit the Remnant, there are no missions available related to Ivan. (I know there's an opportunity to tell the Remnant later on in that mission chain after giving Ivan the keystones, but that feels too late to me, like I've already betrayed the Remnant.)

### Related Issue Links

I didn't see any.

### Desired Solution

Add a mission to talk to some Remnant (maybe a prefect, maybe not) after I know that Ivan wants keystones, but before I give him any. I think it would be neat and (I think) fitting for the Remnant to give me fake keystones to thwart Ivan's research, but it would also work if they tell me to abort the mission or continue with the mission. Really any guidance from the Remnant in that situation would be good.

### Alternative Approaches

Leave it as it is? I'm not sure if this situation bothers anybody other than me.

**Comment by Zitchas** [comment]
Yes, that would definitely be a good thing to have. More missions related to this are planned; and as pointed out, are quire important for various lore, informational, and story reasons. That being said, they are still lower priority than a lot of other elements of the story right now.
